packages/contracts/contracts/L2/messaging/L2StandardBridge.sol

```diff
@@ -146,7 +220,21 @@ contract L2StandardBridge is IL2ERC20Bridge {
             "Can only be called by a the l1TokenBridge"
         );
 
+        // Check to see if the bridge is being used to deposit ETH.
+        // The `msg.value` must match the `_amount` to prevent
+        // ETH from getting stuck in the contract
         if (
+            _l1Token == address(0) &&
+            _l2Token == Lib_PredeployAddresses.OVM_ETH &&
+            msg.value == _amount
+        ) {
+            // An ETH deposit is being made via the Token Bridge.
```

Comment:
What is the worst thing that you can do if you can get the `L2StandardBridge` to call your contract? h/t @smartcontracts

Reply:
reentrency is prevented by the above `require` statement. Another option is to deploy the a `Burner` like contract and then selfdestruct it and then it can transfer the ETH without EVM execution